docs/source/orderer/ordering_service.md

```diff
@@ -352,5 +359,51 @@ blocks `101` to `180`. Blocks `180` to `196` would then be replicated to `R1`
 through the normal Raft protocol.
 
 
+## BFT
+
+For information on how to deploy and manage the BFT orderer, be sure to check out the [deployment guide](../bft_configuration.md).
+
+The protocol used by Fabric's BFT orderer implementation is the [SmartBFT](https://arxiv.org/abs/2107.0692) protocol 
+heavily inspired by the [BFT-SMART](https://www.di.fc.ul.pt/~bessani/publications/dsn14-bftsmart.pdf) protocol,
+which itself can be thought of as a non-pipelined(*) version of the seminal [PBFT](https://pmg.csail.mit.edu/papers/osdi99.pdf) protocol.
+As in Raft, the protocol designates a single leader which batches transactions into a block and sends them to the rest of the nodes, termed followers. 
+However, unlike Raft, the leader is not dynamically selected but is rotated in a round-robin fashion every time the previous leader is suspected of being faulty 
+by the follower nodes.
+
+Further differentiating itself from Raft, where more than half of the nodes are required for the ordering service to be functional, 
+the BFT protocol withstands failures of up to (and not including) a third of the nodes. 
+If a third or more of the nodes crash or are unreachable, no blocks can be agreed upon.
+
+The advantage of the BFT orderer over the Raft orderer is that it can withstand some of the nodes being compromised.
+Indeed, if up to (but not including) a third of the orderer nodes are controlled by a malicous party,
+the system can still accept new transactions, order them, and most importantly ensure the same blocks are committed
+by the rest of the ordering nodes. This is in contrast to Raft, which is not suitable to be deployed in such a harsh adversary model.
+
+Operating the BFT orderer is identicaly to how the Raft orderer is operated: New nodes can be added and removed from
+the channel dynamically and while the system is running, and it is described in the [reconfiguration guide](../create_channel/add_orderer.md).
+
+Similarly to Raft, the BFT leader sends periodical heartbeats to each follower, and if the latter does not hear
+from the leader within a period of time, it starts lobbying other followers to change the leader.
+
+A major difference from Raft, is that when a client submits a transaction to a BFT ordering service, it should send
+the transaction to all nodes instead of to a single node. Sending the transaction to everyone will not only
```

Comment:
It would be good to mention that when using the Fabric Gateway SDKs, the peer Gateway service (../gateway.html) automatically submits the transaction to all ordering service nodes (I think this work has been completed, right?). Otherwise, people will think they need to handle this in their own client applications.

Also, I'd change "everyone" to "all ordering service nodes".

Reply:
Done